packages/react-core/src/components/EmptyState/__tests__/Generated/__snapshots__/EmptyStateIcon.test.tsx.snap

```diff
@@ -10,7 +10,7 @@ exports[`EmptyStateIcon should match snapshot (auto-generated) 1`] = `
     height="1em"
     role="img"
     style="vertical-align: -0.125em;"
-    viewBox="0 0 830 1024"
+    viewBox="0 0 448 512"
```

Comment:
When rebasing with the v5 branch, re-installing node modules and re-running tests, this snapshot was updated, which I would have expected to have been changed in a prior PR.